relay-server/src/metrics_extraction/event.rs

```diff
@@ -1216,66 +1219,50 @@ mod tests {
 
     #[test]
     fn test_app_start_cold_inlier() {
-        assert_eq!(
-            3,
-            extract_span_metrics_op_duration("app.start.cold", 180000.0).len()
-        );
+        let metrics = extract_span_metrics_mobile("app.start.cold", 180000.0);
+        assert!(!metrics.is_empty());
```

Comment:
Should we validate the names of the extracted metrics?

Reply:
Done.